addons/web/static/src/js/core/dom.js

```diff
@@ -548,6 +548,9 @@ var dom = {
      * @return {Promise}
      */
     scrollTo(el, options = {}) {
+        if (!el) {
+            throw new Error("Element missing");
```

Comment:
Comment message was not updated and is now thus wrong "it will just have no effect." -> "it will crash". Also "Element missing" is a bit not enough :wink: